src/browsers/browser.js

```diff
@@ -62,7 +63,7 @@ function launchBrowser(opts) {
     }
 }
 
-function getBrowser(target, dataDir, url) {
+function getBrowser(target, dataDir, chromiumPath, url) {
```

Comment:
Please move `chromiumPath` to the last argument to avoid some unit test failures.